iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/processor/schemachange/RenameDatabaseProcessor.java

```diff
@@ -47,9 +47,14 @@ public class RenameDatabaseProcessor implements PipeProcessor {
   @Override
   public void validate(PipeParameterValidator validator) throws Exception {
     validator.validateRequiredAttribute(PROCESSOR_RENAME_DATABASE_NEW_DB_NAME);
-    newDatabaseName = validator.getParameters().getString(PROCESSOR_RENAME_DATABASE_NEW_DB_NAME);
+    final String database =
+        validator.getParameters().getString(PROCESSOR_RENAME_DATABASE_NEW_DB_NAME);
+    if (database == null) {
+      throw new PipeException("new database name cannot be null");
+    }
```

Comment:
No need. Because of `validator.validateRequiredAttribute(PROCESSOR_RENAME_DATABASE_NEW_DB_NAME);`